src/raw/config-avalanche.js

```diff
@@ -0,0 +1,201 @@
+      symbol: "2AVAX",
+      launchTimestamp: 1639490400,
+      createdTimestamp: 1638805641,
+      yieldManager: {
+        address: "0x621cda10820555adAe8506eeC296cB9621E491Ff",
+        description: "Aave-based yield manager",
+        codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/YieldManagers/YieldManagerAaveBasic.sol",
+      },
+      oracle: {
+        decimals: 8,
+        heartbeat: 135,
+      },
+      longToken: {
+        address: "0xa43a73f17ECde987A9127A5cBE46B7923F351c32",
+        symbol: "flAvax2",
+        iconUrl:
+          "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/icons/tokens/avax/avax_long.png",
+      },
+      shortToken: {
+        address: "0xdC883B026F78FF494199778001b23f38edA89d6d",
+        symbol: "fsAvax2",
+        iconUrl:
+          "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/icons/tokens/avax/avax_short.png",
+      },
+    },
+    {
+      index: 2,
+      name: "JOE Market",
+      description:
+        "JOE is a governance token that rewards its holders with a share of exchange revenues from the Trader Joe protocol. Trader Joe is a one-stop decentralized trading platform on the Avalanche network.",
+      oracleHeartbeatInSec: 1800,
+      iconUrl: "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/tokens/joe/joe.png",
+      oracleDecimals: 8,
+      leverage: 2,
+      paymentTokenSymbol: "DAI",
+      symbol: "2JOE",
+      launchTimestamp: 1639663200,
+      createdTimestamp: 1639643931,
+      yieldManager: {
+        address: "0x47a21F14794b6229cc2a1ddfe4498C9e48f1C16c",
+        description: "Aave-based yield manager that uses JOE as backend instead of Aave.",
+        codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/YieldManagers/YieldManagerAaveBasic.sol",
+      },
+      longToken: {
+        iconUrl:
+          "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/tokens/joe/joe_long.png",
+        address: "0x6A621D256CFEDa1c10ab0Cbd1Ff8d5310b35e4d3",
+        symbol: "fl2JOE",
+      },
+      shortToken: {
+        iconUrl:
+          "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/tokens/joe/joe_short.png",
+        address: "0x1dCAA44bEA82bd135C51b158E5E702e3C1843951",
+        symbol: "fs2JOE",
+      },
+    },
+    {
+      index: 3,
+      name: "QI Market",
+      description:
+        "QI is the governance token for BENQI, Avalanche's leading liquidity protocol. QI is earned as a reward for supplying liquidity in BENQI, and can be supplied in their protocol for a tasty APY.",
+      oracleHeartbeatInSec: 1800,
+      icon: "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/tokens/qi/qi.png",
+      oracleDecimals: 8,
+      leverage: 2,
+      paymentTokenSymbol: "DAI",
+      symbol: "2QI",
+      launchTimestamp: 1642514400,
+      createdTimestamp: 1642511122,
+      usesBenqiForYield: true,
```

Comment:
probably want to remove`usesBenqiForYield` and `usesJoeForYield`. also `oracleDecimals` i.e. everything not in the config shape.

Reply:
Ah damn I thought I removed those. Thanks for catching